docs/building_and_running_tests.rst

```diff
@@ -0,0 +1,38 @@
+.. _building_and_running_tests:
+
+Building and Running Tests
+==========================
+
+The functional are tests are enabled by default, and can be enabled/disabled
```

Comment:
```suggestion
The functional tests are enabled by default, and can be enabled/disabled
```

Reply:
Done in https://github.com/oneapi-src/oneMKL/pull/510/commits/4f6af02e17e392695cebf7b84af6a7a76ebb5d6f.